### PR TITLE
Demos & docs for VPT

### PR DESCRIPTION
**Pull request by sovrasov**
- Samples for VPT
- Docs
- Refine ZSL VPT API

### CODE REVIEW

File: examples/python/visual_prompting/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+## Run example
+To run the example, please execute the following command:
+```bash
+python run.py <path_to_image> <encoder_path> <decoder_path> <prompts>
+```
+where prompts are in X Y format.
+
+To run the pipeline out-of-the box you can download the test data by running the following command from the repo root:
+```bash
+python tests/python/accuracy/prepare_data.py -d data
```
Comment (eunwoosh) [inline examples/python/visual_prompting/README.md:20]:
```suggestion
pip install httpx
python tests/python/accuracy/prepare_data.py -d data
```
It seems that httpx is necessary but it isn't installed when installing model_api

File: examples/python/zsl_visual_prompting/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+## Run example
+To run the example, please execute the following command:
+```bash
+python run.py <path_to_source_image> <path_to_target_image> <encoder_path> <decoder_path> <prompts> -t <mask confidence threshold>
+```
+where prompts are in X Y format.
+
+To run the pipeline out-of-the box you can download the test data by running the following command from the repo root:
+```bash
+python tests/python/accuracy/prepare_data.py -d data
```
Comment (eunwoosh) [inline examples/python/zsl_visual_prompting/README.md:21]: Same as above.

File: examples/python/zsl_visual_prompting/README.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# Zero-shot Segment Anything example
+This example demonstrates how to use a Python API implementation of Segment Anything pipeline inference:
```
Comment (eunwoosh) [inline examples/python/zsl_visual_prompting/README.md:2]:
```suggestion
This example demonstrates how to use a Python API implementation of Zero-shot Segment Anything pipeline inference:
```
I'm not sure it's better, so please ignore it if you think it isn't :)

File: examples/python/zsl_visual_prompting/README.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+To run the pipeline out-of-the box you can download the test data by running the following command from the repo root:
+```bash
+python tests/python/accuracy/prepare_data.py -d data
+```
+and then run
+```bash
+python run.py ../../../data/coco128/images/train2017/000000000025.jpg \
+     ../../../data/coco128/images/train2017/000000000072.jpg ../../../data/otx_models/sam_vit_b_zsl_encoder.xml \
+     ../../../data/otx_models/sam_vit_b_zsl_decoder.xml 464 202 -t 0.7
```
Comment (eunwoosh) [inline examples/python/zsl_visual_prompting/README.md:27]: Maybe it's better to explain what `-t` is and how threshold works?

File: model_api/python/model_api/models/utils.py [hunk 5]
```diff
@@ -164,6 +166,7 @@ def __str__(self) -> str:
 class PredictedMask(NamedTuple):
     mask: list[np.ndarray]
     points: list[np.ndarray] | np.ndarray
+    scores: list[float] | np.ndarray
```
Comment (eunwoosh) [inline model_api/python/model_api/models/utils.py:169]: How about updating `__str__` method to include `scores` information?

**Review by eunwoosh — COMMENTED** [review]
Thanks for your work :) I left some comments. Please take a look.

File: model_api/python/docs/action_classification.md [hunk 6]
```diff
@@ -1,10 +1,53 @@
...
+
+A single tensor containing softmax-activated logits.
+
+## Wrapper input-output specifications
+
+### Inputs
+
+A single clip in THWC format.
+
+## Outputs
```
Comment (eunwoosh) [inline model_api/python/docs/action_classification.md:47]:
```suggestion
### Outputs
```
It seems typo.

File: examples/python/visual_prompting/README.md (hunk 1 above)
Comment (sovrasov) [reply examples/python/visual_prompting/README.md:21]: Added

File: examples/python/zsl_visual_prompting/README.md (hunk 3 above)
Comment (sovrasov) [reply examples/python/zsl_visual_prompting/README.md]: Added

File: model_api/python/model_api/models/utils.py (hunk 5 above)
Comment (sovrasov) [reply model_api/python/model_api/models/utils.py:169]: Added

File: examples/python/zsl_visual_prompting/README.md (hunk 4 above)
Comment (sovrasov) [reply examples/python/zsl_visual_prompting/README.md:29]: Added

**Review by eunwoosh — APPROVED** [review]
